### PR TITLE
Add `FRAC_1_SQRT_2PI` constant to f16/f32/f64/f128

### PR DESCRIPTION
**Pull request by sunsided**
This adds the `FRAC_1_SQRT_2PI` to the `f16`, `f32`, `f64` and `f128` as [`1/√(2π)`](https://www.wolframalpha.com/input?i=1%2Fsqrt%282*pi%29). The rationale is that while `FRAC_1_SQRT_PI` already exists, [Gaussian calculations](https://en.wikipedia.org/wiki/Gaussian_function) for random normal distributions require a `1/(σ√(2π))` term, which could then be directly expressed e.g. as `f32::FRAC_1_SQRT_2PI / sigma`.

The actual value is approximately `1/√(2π) = 0.3989422804014326779399460599343818684758586311649346576659258296…`. Truncated/rounded forms were used for the individual types. 

---

~~I did not any of the `#[unstable]` attributes since I am not aware of their implications.~~

**Edit:** I applied the stability attributes from the surrounding types according to what seemed most likely correct. I believe the `more_float_constants` feature marker is incorrectly applied, but I wasn't sure how to proceed.

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
New unstable API surface area currently requires [an ACP](https://github.com/rust-lang/libs-team/issues/new?assignees=&labels=api-change-proposal%2C+T-libs-api&projects=&template=api-change-proposal.md&title=%28My+API+Change+Proposal%29). Please file one and if it's approved we can merge.

@rustbot author

**Comment by sunsided** [comment]
Thanks, added it at https://github.com/rust-lang/libs-team/issues/383. Exciting!

@rustbot review

File: library/core/src/num/f128.rs [hunk 1]
```diff
@@ -68,6 +68,11 @@ pub mod consts {
     pub const FRAC_1_SQRT_PI: f128 =
         0.564189583547756286948079451560772585844050629328998856844086_f128;
 
+    /// 1/sqrt(2π)
+    #[unstable(feature = "f128", issue = "116909")]
```
Comment (tgross35) [inline library/core/src/num/f128.rs:72]: For this and f16, please also add a comment to note there is a second feature gate:

```rust
#[unstable(feature = "f128", issue = "116909")]
// #[unstable(feature = "more_float_constants", issue = "103883")]
pub const FRAC_1_SQRT_2PI: /* ... */
```

**Comment by sunsided** [comment]
I added `#[doc(alias = "FRAC_1_SQRT_TAU")]` to the constants to create a bit of symmetry with the already-existing `TAU` constant, as discussed in rust-lang/libs-team#383.

**Pull request merged**